usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -94,15 +98,13 @@ def handle(self, *args, **options):
         loader = ThreadedDataLoader(Procurement, field_map=field_map, value_map=value_map)
         loader.load_from_file(options['file'][0])
 
-    def get_agency(self, row):
-        agency = Agency.objects.filter(subtier_code=self.get_agency_code(row['maj_agency_cat'])).first()
+    def get_agency(self, agency_string):
+        agency = Agency.objects.filter(subtier_code=self.get_agency_code(agency_string)).first()
```

Comment:
Yeah, agreed that if we're guaranteeing uniqueness at the db level, it might be overkill to put in try/catch all over the place. Will keep it as a watch point going forward, thinking more about where it makes sense.